### ISSUE TITLE
Master branch not compiling

### ISSUE THREAD

**Issue by apupier** [jboss-fuse/fuse-apicurito-generator#16]
with

`[ERROR] Failed to execute goal org.jboss.redhat-fuse:fabric8-maven-plugin:7.2.0.fuse-720009:resource (fmp) on project fuse-apicurito-generator: Execution fmp of goal org.jboss.redhat-fuse:fabric8-maven-plugin:7.2.0.fuse-720009:resource failed: Plugin org.jboss.redhat-fuse:fabric8-maven-plugin:7.2.0.fuse-720009 or one of its dependencies could not be resolved: Failed to collect dependencies at org.jboss.redhat-fuse:fabric8-maven-plugin:jar:7.2.0.fuse-720009 -> io.fabric8:fabric8-maven-plugin:jar:3.5.33.fuse-720013: Failed to read artifact descriptor for io.fabric8:fabric8-maven-plugin:jar:3.5.33.fuse-720013: Could not transfer artifact io.fabric8:fabric8-maven-plugin:pom:3.5.33.fuse-720013 from/to jb-eap-7.1-brew (http://download.eng.bos.redhat.com/brewroot/repos/jb-eap-7.1-rhel-7-maven-build/latest/maven): download.eng.bos.redhat.com: Name or service not known: Unknown host download.eng.bos.redhat.com: Name or service not known -> [Help 1]`

see https://circleci.com/gh/jboss-fuse/fuse-apicurito-generator/24

**Comment by apupier** [comment]
so the choice has been made to update the fuse.version every time the staging version vanished? https://github.com/jboss-fuse/fuse-apicurito-generator/pull/20/files#diff-600376dffeb79835ede4a0b285078036R33

**Comment by EricWittmann** [comment]
Typically I would suggest NOT depending on a staging version, at least not on `master`.  But if that's necessary then what other options are there?

**Comment by bfitzpat** [comment]
Agreed. The project was initially built against a staging version - and there isn't a production version yet -- we really don't have a choice. We can ask @janstey to stash 7.2.0.fuse-720018 for now like we have done with wsdl2rest?

**Comment by apupier** [comment]
some options:
- ask to runtime fuse-engineering team to release a version
- ask to stash a version with required dependencies
- see how fuse-engineering runtime is handling their staging versions, do they have automatic update? (even if I hate that). is it what their "pipeline" is doing? In this case, should this project be part of the pipeline?
- have an alert when a staging version is removed and update it manually
- accept to have master build that can be broken and be ready to provide new PR to update when it occurs

**Comment by cunningt** [comment]
Another option - on each successful build (we average maybe a couple a week), I can automatically have the build generate a PR to keep this project up-to-date with the most recent build versions.    In the case of fuse-apicurito-generator, it looks like that'd mean updating camel.version and fuse.version.
